apps/devhub/tests/test_views.py

```diff
@@ -1165,6 +1168,8 @@ def test_step_2_with_cookie(self):
 
     def test_step_2_no_cookie(self):
         # We require a cookie that gets set in step 1.
+        self.user.update(read_dev_agreement=None)
+
         r = self.client.get(reverse('devhub.submit.2'), follow=True)
         self.assertRedirects(r, reverse('devhub.submit.1'))
```

Comment:
because the submit view has a new feature where it skips the agreement if already read, you need to add a test for that. It would look something like this:
````
self.user.update(read_dev_agreement=now())
res = self.client.get(reverse('devhub.submit.1'), follow=True)
self.assertRedirects(res, reverse('devhub.submit.2'))
````